### ISSUE TITLE
Transactions don't work in interactive console

### ISSUE THREAD

**Issue by rtsisyk** [tarantool/tarantool#869]
```
tarantool> _ = box.space.test:create_index('primary')

---
...

tarantool> box.begin()

---
...

tarantool> box.space.test:insert{1, 'tx test'}

---
- [1, 'tx test']
...

tarantool> box.space.test:get{1}

---
...
```

I understand why it happens, but it is not obvious for users.

**Comment by R-omk** [comment]
This has probably become possible since `memtx_use_mvcc_engine` option was introduced.

**Comment by kyukhin** [comment]
> This has probably become possible since `memtx_use_mvcc_engine` option was introduced.

Yes, it is. Implemented in ver 2.6. See the [issue](https://github.com/tarantool/tarantool/issues/4897).
